acceptance/suites/tests/authorization/hup_reloads_crl.rb

```diff
@@ -0,0 +1,68 @@
+test_name '(SERVER-1380) CRL should reload after a server HUP' do
+
+server = master.puppet['certname']
+
+teardown do
+  # Leave agents with working certs. Destroy the revoked cert on the master;
+  # remove all traces on the agent; make the agent request a shiny new cert; 
+  # have the master sign it; then use it to make sure it works.
+  agents.each do |a|
+    if not_controller(a)
+      hostcert_file = a.puppet['hostcert']
+      reqdir = a.puppet['requestdir']
+      nodefqdn = a.hostname
+      request_file = File.join(reqdir, nodefqdn + ".pem")
+
+      # FIXME: this is a dirty nasty hack that Beaker should handle seamlessly
```

Comment:
It was mostly a whing that I had to write the code myself. I think I added this comment at the moment I realized I'd spent more time writing the `teardown` block than I spent writing then rest of the test. 

>